settings.gradle.kts

```diff
@@ -1,6 +1,9 @@
+rootProject.name = "LokateCMP"
```

Comment:
The project has both SDK and an example app(composeApp); CMP naming may be misleading since the SDK does not use Compose.